### ISSUE TITLE
Change codec from H.265 to H.264

### ISSUE THREAD

**Issue by felipecrs** [NeiroNx/python-dvr#55]
Simply changing like following isn't working:

```
enc_info = cam.get_info("Simplify.Encode")
print("Encoder config:", enc_info)

# Change H.265 to H.264
enc_info[0]['MainFormat']['Video']['Compression'] = 'H.264'

# Apply changes
cam.set_info("Simplify.Encode", enc_info)

print("New encoder config:", cam.get_info("Simplify.Encode"))
```

The output still shows `H.265`. When I open the camera web interface, there is no option to change the codec too:

![image](https://github.com/NeiroNx/python-dvr/assets/29582865/3296700b-f606-4aa1-ba16-edc0dc81d2c0)

I am wondering if there is something I can do to change the codec to H.264. Any help would be greatly appreciated.

**Comment by widgetii** [comment]
According to your screenshot, there is no such option on the camera

**Comment by felipecrs** [comment]
Yes, there is not and that must be why it did not work. But I was wondering if I could perhaps create such option using `python-dvr`. If you believe this is totally undoable, I can close this issue.

**Comment by widgetii** [comment]
Why don't you ask the manufacturer of your camera about codecs supported?

**Comment by felipecrs** [comment]
Yes, I actually did that. BTW, this is the camera: https://aliexpress.com/item/1005005731238053.html

It seems based on XM530. And apart from the unavailability of H.264 codec, this camera is amazing for the price. It even supports PTZ presets.

I'm still waiting for their response though, and I find very unlikely that they will address my concern.
